README.md

```diff
@@ -92,6 +94,9 @@ Some of the rules can be configured from your `apostrophe-seo` declaration.
         max: 80
       },
       minWordsOnPage: 400
+    },
+    headers: {
```

Comment:
I think this has to be `scanHeaders` right?

Reply:
Right, thanks